### ISSUE TITLE
Add arial-label to links "en" -> "english"

### ISSUE THREAD

**Issue by fatcrobat** [terminal42/contao-changelanguage#155]
Screenreader will not properly output for text links with "EN", "CS", "DE". Best practice here is to add an aria-label that replaces the text links. 

Before:

```
<a href="app_dev.php/en/" title="Englisch" class="lang-en nofallback first" hreflang="en" lang="en" itemprop="url"><span itemprop="name">EN</span></a>
```

After: 

```
<a href="app_dev.php/en/" title="Englisch" class="lang-en nofallback first" hreflang="en" lang="en" itemprop="url" aria-label="Englisch"><span itemprop="name">EN</span></a>
```

Labels should use the `$GLOBALS['TL_LANG']['LNG']` language keys.
This is part of the result of an BITV/WCAG-Test we have done for one of our web sites by BIK.

Current workaround is to modify the navigation template and add the aria-label by your own:

```
<a href="app_dev.php/en/" title="<?= $GLOBALS['TL_LANG']['LNG'][$item['item']->getRootPage()->language]; ?>" class="lang-en nofallback first" hreflang="en" lang="en" itemprop="url" aria-label="<?= $GLOBALS['TL_LANG']['LNG'][$item['item']->getRootPage()->language]; ?>"><span itemprop="name">EN</span></a>
```
The title attribute may also use the `aria-label` value.

**Comment by aschempp** [comment]
There is already a title attribute that contains this value, and as far as I know the title is preferred by screen readers?

**Comment by fatcrobat** [comment]
The title by default contains the title of the root page. This is in most use cases not the description for the localized language name and in can not be localized. 

The title works also, but should be more meaningful for the screenreader. Alternative is to use the aria-label and translate visible language automatically.

**Comment by aschempp** [comment]
if I understand correctly, the title of the root page does not really make sense ever? So maybe we should just update the title to contain the language name instead of the root page?

**Comment by aschempp** [comment]
I'm closing this since I'm unsure what the best solution is. Let's reopen if there is more information.
